### ISSUE TITLE
[!] VoiceOver/screen reader labeling/accessibility is broken in new UI design

### ISSUE THREAD

**Issue by rfiorentino1** [oznu/homebridge-config-ui-x#1946]
### Describe The Bug

Ever since the new UI design changes, VoiceOver on Mac OS or IOS does not read the labels for plugin updates, settings buttons, etc. Also, this was a previous bug but still exists, the screen reader cannot see anything in the logs or the terminal text boxes within the UI. If someone can Point me to the source HTML etc that is creating these things I can fix it :) 

Also, there is now an unlabeled button at the top of the status page, and a button labeled "lock" which I'm not sure what the function is.

### Homebridge UI Version

4.54.1

### Homebridge Version

1.7.0

### Node.js Version

18

### Operating System

Synology DSM

### Environment Info

- [ ] Using Docker?
- [ ] Using Hyper-V?
- [X] Using hb-service?

### Raspberry Pi Model

None

**Comment by bwp91** [comment]
Hi @rfiorentino1 

This is something we should absolutely get sorted out before any further release.

I don't suppose you are on the Homebridge Discord server? Would be interesting to learn more about further stuff we could do in the UI to assist.

Cheers!

**Comment by rfiorentino1** [comment]
Hi Ben,
Thanks so much for replying so quickly, I really appreciate it. I'm not on the discord, but I definitely could be, though I need to get familiar with it again because it's been quite a while since I've played around with discord. Is there a specific channel you want me to join?

**Comment by bwp91** [comment]
Hopefully this invite link works
https://discord.gg/C87Pvq3

And feel free to find my username bwp91 in the user list and send me a dm!
